.github/workflows/test_generation.yml

```diff
@@ -17,22 +18,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9]
```

Comment:
makes sense, we should also officially drop python 3.8 in https://github.com/huggingface/optimum-intel/blob/v1.20.1/setup.py#L91

Reply:
dropped from setup.py